### PR TITLE
Site updates

### PR DESCRIPTION
**Pull request by blacktm**
Working on some updates to the dev docs. Site can be previewed at: https://federalist-proxy.app.cloud.gov/preview/18f/identity-dev-docs/site-updates

### CODE REVIEW

**Comment by s2t2** [comment]
Hey @blacktm one thing I notice is that the certs page went away? Is there alternative guidance on how to generate the certs/keys?

File: _pages/attributes.md [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
-| Social security number | `ssn`         | `social_security_number` |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
-| Phone                  | `phone`       | `phone`                  |       | <img src="{{ site.baseurl}}/assets/img/check.svg" alt="checkmark"> |
+| Attribute              | LOA1 | LOA3  | OpenID Connect | SAML |
+| ---------------------- | ---- | ----- | -------------- | ---- |
+| [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier) | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> | `sub` | `uuid` |
+| Email                  | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> | `email` | `email` |
+| First name             |      | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> | `given_name`             | `first_name`  |
+| Middle name            |      | <img src="{{ site.baseurl }}/assets/img/check.svg" alt="checkmark"> |                          | `middle_name` |
```
Comment (andrewhughey) [inline _pages/attributes.md]: I don't think we capture middle name anywhere

Comment (blacktm) [reply _pages/attributes.md]: Oh, how about that. Thanks @andrewhughey!

**Comment by blacktm** [comment]
@s2t2 Yea, we could keep guidance for how to create certs (public/private keys) for testing. I wasn't actually sure it was useful, since people probably know how to do this. Also, there's a lot of stuff on the web already, [like this](https://www.ibm.com/support/knowledgecenter/en/SSWHYP_4.0.0/com.ibm.apimgmt.cmc.doc/task_apionprem_gernerate_self_signed_openSSL.html).

**Comment by blacktm** [comment]
Ok, I think we're ready to merge this. Final call for changes this iteration.

**Pull request merged**